script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -0,0 +1,154 @@
+
+function main {
+  # create array with network/s for which the script should be executed
+  local NETWORKS=()
+
+  # loop through networks list and add each network to ARRAY that is not excluded
+  # while IFS= read -r line; do
+  #   NETWORKS+=("$line")
+  # done <"./networks"
+  NETWORKS=("arbitrum" "polygon" "optimism") # TODO: remove <<<<<<<<<---------------------------------------------------------------------------------------- (WILL MAKE SURE THAT THE TEST RUNS ONLY ON THREE NETWORKS)
+
+  echo "networks found: ${NETWORKS[@]}"
```

Shellcheck

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)